app/views/layouts/session_answers.html.erb

```diff
@@ -10,4 +10,6 @@
   </div>
 </main>
 
+<%= @form.session[:session_answers].inspect %>
```

Comment:
That's debugging code. I don't think it should be committed.

Reply:
Fixed - thanks 👍